### PR TITLE
calculate pitch

### PR DESCRIPTION
**Pull request by aishinchi**
Add post-process pitch features to athena.transform. Default params produce 3-dimensional pitch features consisting of (pov-feature, pitch-feature, delta-pitch-feature).

### CODE REVIEW

File: athena/data/datasets/speech_recognition.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ class SpeechRecognitionDatasetBuilder(BaseDatasetBuilder):
         "input_length_range": [20, 50000],
         "output_length_range": [1, 10000],
         "speed_permutation": [1.0],
+        "vocab_file": "examples/asr/hkust/data/vocab",
```
Comment (tjadamlee) [inline athena/data/datasets/speech_recognition.py]: This should not be included in current commit ?

**Review by tjadamlee — CHANGES_REQUESTED** [review]
Please only submit the necessary change related to the pitch

**Comment by tjadamlee** [comment]
@Some-random @leixiaoning 
please help test this

**Comment by Some-random** [comment]
> @Some-random @leixiaoning
> please help test this

Running HKUST benchmark again with 80 dim FBANK + 3 dim Pitch right now

**Comment by tjadamlee** [comment]
@aishinchi @Some-random 
if currently it is not finished, please close this pr
You can re push after evaluated

**Comment by tjadamlee** [comment]
please check, and then re push

**Pull request closed without merging**